lib/systems/default.nix

```diff
@@ -89,6 +89,10 @@ rec {
         # is why we use the more obscure "bfd" and not "binutils" for this
         # choice.
         else                                     "bfd";
+      # The standard lib directory name on *other* implementations of this system, not nixpkgs'
+      libDir = if final.isLinux && (final.isx86_64 || final.isMips64 || final.isPower64)
```

Comment:
Is lib64 not used on e.g. 64-bit RISC-V?

Reply:
It should only be necessary on cpus which can run multiple different instruction sets, making disambiguation in userspace necessary. iirc 64-bit risc-v doesn't do that. Anyway, I pulled that list from `nix-ld`.

Reply:
In that case I think the comment could be improved — it would be good if it was more precise than "*other* implementations" ("that binaries distributed for this platform will commonly use"?)